src/main/java/mate/academy/bookstorev2/dto/category/CreateCategoryRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.bookstorev2.dto.category;
+
+import jakarta.persistence.Column;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateCategoryRequestDto {
+    @Column(nullable = false)
```

Comment:
```suggestion
    @NotBlank
```

Also, you can validate Size for each property